grizzly/args.py

```diff
@@ -52,14 +51,16 @@ def __init__(self):
                 formatter_class=SortingHelpFormatter, conflict_handler="resolve"
             )
 
+        targets = scan_plugins("grizzly_targets")
+        if not targets:
+            self.parser.error("No Platforms (Targets) are installed")
+
         self.parser.add_argument("binary", help="Firefox binary to run")
         self.parser.add_argument(
             "--log-level",
+            choices=sorted(self._level_map),
             default="INFO",
-            help="Configure console logging. Options: %s (default: %%(default)s)"
-            % ", ".join(
-                k for k, v in sorted(self._level_map.items(), key=lambda x: x[1])
-            ),
+            help="Configure console logging (default: %(default)s)",
```

Comment:
You could use `"Configure console logging. Options: %(choices)s (default: %(default)s)"` to keep the old format.

Reply:
But wouldn't that be redundant?

We now get the choices automatically:
```
--log-level {CRIT,DEBUG,ERROR,INFO,WARN}
                      Configure console logging (default: INFO)
```

Reply:
you're right